frontend/src/components/UserProfilePanels/UserInfo.css

```diff
@@ -0,0 +1,31 @@
+.profilePic {
+  width: 150px;
+  border-radius: 50%;
+}
+
+.user-edit-button {
+  border: none;
+  margin: 10px;
+}
+
+.user {

```

Comment:
I wonder if this one, `.profilePic`, and maybe `.user-name` may end up being a little too generic in terms of class names, since CSS is shared across the site. There's at least one other place we'll have a profile pic displayed (in the header), and I worry about this one potentially colliding with later work, as well.

Reply:
good point, I can edit these to be more specific!